tests/centroid.py

```diff
@@ -137,11 +137,7 @@ class SingleFrameMeasurementTaskTestCase(utilsTests.TestCase):
     def mySetup(self, runCentroider=True):
         msConfig = measBase.SingleFrameMeasurementConfig()
         msConfig.algorithms.names = ["base_SdssCentroid"]
-        if not runCentroider:
-            msConfig.slots.centroid = None
-        else:
-            msConfig.slots.centroid = "base_SdssCentroid"
-
+        msConfig.slots.centroid = "base_SdssCentroid"
```

Comment:
You've eliminated the only use of the `runCentroider` argument. Drop it entirely?